aspnetcore/fundamentals/openapi/samples/9.x/BuildTime/Skip.cs

```diff
@@ -0,0 +1,34 @@
+#if NEVER
+#region snippet_1
+using ControllerApi.Data;
+using Microsoft.EntityFrameworkCore;
+using System.Reflection;
+
+var builder = WebApplication.CreateBuilder(args);
+
+if (Assembly.GetEntryAssembly()?.GetName().Name != "MyTestApi")
+{
+    builder.Services.AddDbContext<ControllerApiContext>(options =>
+        options.UseSqlServer(builder.Configuration.GetConnectionString("ControllerApiContext")
+        ?? throw new InvalidOperationException("Connection string 'ControllerApiContext' not found.")));
+}
+
```

Comment:
@martincostello  can you provide the code to demonstrate  *To prevent these code paths from being invoked by the build-time generation pipeline, they can be conditioned behind a check of the entry assembly:* ?

Reply:
@Rick-Anderson I can take care of providing the code sample for this. I'll have to take a look at it next week.